### PR TITLE
update long pressed UI on messages

### PR DESCRIPTION
**Pull request by chungwwei**
👋  Mux, I made an update to the long pressed UI on messages, accept it if you find it worthy. :)

### CODE REVIEW

File: lib/components/chat_panel.dart [hunk 1]
```diff
@@ -54,35 +80,96 @@ class _ChatPanelWidgetState extends State<ChatPanelWidget> {
...
+                                        title: Text('${message.author}',
+                                            style: TextStyle(fontSize: 20)),
+                                        tileColor: Colors.blueAccent,
+                                        trailing: CloseButton(),
+                                      ),
+                                      ListTile(
+                                        leading: Icon(Icons.favorite,
+                                            color: Colors.deepPurpleAccent),
+                                        title:
+                                            Text('Following Since Jan 11 2024'),
```
Comment (kevmo314) [inline lib/components/chat_panel.dart:108]: Can this data be fetched from the API instead of hardcoded?

File: lib/components/chat_panel.dart [hunk 2]
```diff
@@ -54,35 +80,96 @@ class _ChatPanelWidgetState extends State<ChatPanelWidget> {
...
-                                    final model = Provider.of<UserModel>(
-                                        context,
-                                        listen: false);
-                                    model.unban(
-                                        model.channels.first, message.author);
-                                    Navigator.pop(context);
-                                  }),
-                            ]),
-                          );
+                              shape: RoundedRectangleBorder(
```
Comment (kevmo314) [inline lib/components/chat_panel.dart:83]: Leave out custom shapes, the material defaults usually look the most consistent.

File: lib/components/chat_panel.dart [hunk 3]
```diff
@@ -54,35 +80,96 @@ class _ChatPanelWidgetState extends State<ChatPanelWidget> {
...
+                                  borderRadius:
+                                      BorderRadius.all(Radius.circular(20.0))),
+                              child: Wrap(
```
Comment (kevmo314) [inline lib/components/chat_panel.dart:86]: What does `Wrap` do here?

File: lib/components/chat_panel.dart [hunk 4]
```diff
@@ -54,35 +80,96 @@ class _ChatPanelWidgetState extends State<ChatPanelWidget> {
...
+                                children: [
+                                  ListView(
+                                    shrinkWrap: true,
+                                    children: [
+                                      ListTile(
+                                        shape: RoundedRectangleBorder(
+                                            borderRadius: new BorderRadius.only(
```
Comment (kevmo314) [inline lib/components/chat_panel.dart:93]: Omit `new`, it's not necessary in the latest versions of dart.

File: lib/components/chat_panel.dart (hunk 1 above)
Comment (chungwwei) [reply lib/components/chat_panel.dart:108]: Looks like the data can be fetched from the API, Link: [https://dev.twitch.tv/docs/api/reference#get-users-follows](https://dev.twitch.tv/docs/api/reference#get-users-follows). But I had no experience with pulling data from an API with dart.

File: lib/components/chat_panel.dart (hunk 3 above)
Comment (chungwwei) [reply lib/components/chat_panel.dart:86]: It's not useful in the layout. I was playing with Wrap and Column widgets, forgot to remove it. Will remove.

**Pull request closed without merging**